packages/platform-ios/src/commands/runIOS/index.js

```diff
@@ -214,12 +227,11 @@ async function runOnDevice(selectedDevice, scheme, xcodeProject, args: FlagsT) {
   );
 
   if (iosDeployOutput.error) {
-    logger.error(
-      '** INSTALLATION FAILED **\nMake sure you have ios-deploy installed globally.\n(e.g "npm install -g ios-deploy")',
-    );
-  } else {
-    logger.info('** INSTALLATION SUCCEEDED **');
+    throw new Error(`Failed to install the app on the device. We've encountered an error in "ios-deploy" command.
+      Here are more details: ${isIOSDeployInstalled.error.message}`);
   }
+
+  return logger.info('App installed on device.');
```

Comment:
```suggestion
  return logger.success('Installed the app on the device');
```

Reply:
Any reason for omitting the `.`? good point on `success` ✅

Reply:
Nope, let's leave it. Would be cool to set a convention on when we set a dot and when not :D